projects/core/components/svg/svg.component.ts

```diff
@@ -46,10 +50,13 @@ export class TuiSvgComponent {
     private readonly isIE = isIE(this.userAgent);
     private icon = '';
 
+    @Input()
+    processor: TuiStringHandler<string> = this.globalProcessor;
```

Comment:
do we need `@Input()` if we already have DI variable ?

I think DI is enough (user can override processor on any level of DI scope).

Reply:
fixed